app/services/api/v1/ecf/participants_query.rb

```diff
@@ -17,6 +17,7 @@ def induction_records
           scope = InductionRecord
                     .select(*necessary_fields)
                     .eager_load(:schedule)
+                    .joins(induction_programme: { partnership: { lead_provider: %i[cpd_lead_provider] } })
```

Comment:
I'd like to have a think about this as we already join on the latest induction record so need to join on this again as we have access to the lead provider already, will wait till we agree on the structure of the table first